code/plot_gii_surface.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+""" 
+
+Example usage:
+python plot_gii_surface.py /path/to/my/mesh /path/to/my/map.gii \
+                          <hemisphere> /path/to/my/img.png \
+                          <view>
+
+"""
+
+import nilearn
+from nilearn import plotting
```

Comment:
you can do either `import nilearn` or `from nilearn import plotting`, but you don't need both here. If you do the first one, leave line 30 as-is.  If you do the second, you can remove the `nilearn.` from line 30.